frontend/ui/widget/focusmanager.lua

```diff
@@ -219,7 +219,10 @@ end
 
 --- Container call this method after init to let first widget render in focus style
 function FocusManager:focusTopLeftWidget()
-    self:onFocusMove({0, 0})
+    if Device:hasDPad() then
```

Comment:
```suggestion
    --- @todo Temporary workaround to prevent showing focus on touch-only devices, see <https://github.com/koreader/koreader/pull/8712#issuecomment-1023134610>
    if Device:hasDPad() then
```

Reply:
Not sure it's a "temporary workaround", seems like just the right way to do it :)
(I mentionned "quick fix" as "fix this quick!", but it looks like the real fix, not like a workaround :)

Reply:
Please ignore then. But yes, I interpreted "as a quick fix" as in let's fix this now and worry about fixing it correctly later.